doc/operations/configuration.rst

```diff
@@ -0,0 +1,235 @@
+Configuration
+=============
+
+The configuration for each CCF node must be contained in a single JSON configuration file specified to the ``cchost`` executable via the ``--config </path/to/configuration/file>`` argument.
+
+.. tip:: JSON configuration samples:
+
+    - Minimal configuration: https://github.com/microsoft/CCF/blob/main/samples/config/minimal_config.json
+    - Full configuration: https://github.com/microsoft/CCF/blob/main/samples/config/config.json
+
+Configuration Options
+---------------------
+
+``enclave_file``
+~~~~~~~~~~~~~~~~
+
+Path to CCF enclave application.
+
+``enclave_type``
+~~~~~~~~~~~~~~~~
+
+Type of enclave application (either ``release``, ``debug`` or ``virtual``). Default value: ``release``.
```

Comment:
Every time I see several `foo_(a|b|..)`, I wonder if we shouldn't have a section `foo` with `foo.a`, `foo.b`. On the one hand, it seems cleaner and more future proof (can add fields without crowding the top level), but on the other hand, how much nesting is too much nesting? I genuinely do not know.